### DISCUSSION TITLE
bigger binary

### DISCUSSION THREAD

**Discussion by jjirasek** [mathieucarbou/ESPAsyncWebServer#33] [Q&A]
Hello,
since version 2.10.5 I have about 6kB bigger binary, is this OK? btw thank you for work.

**Comment by mathieucarbou** [comment]
Changes in 2.10.5 only impact ESP32 on Arduino 2 where the mbedtls dependency was removed (replaced by an internal SHA1 class).

The only case where you can have a size increase is if you still use arduino 2 with esp32 and another part of your code or libs depend on mbedtls. In this case, the added sha1 class adds up to the size but 6kB is too much for it.

Look at the changes: https://github.com/mathieucarbou/ESPAsyncWebServer/compare/v2.10.4...v2.10.5

You can use the PIO memory analyser to help you on that.

**Comment by jjirasek** [comment]
I am using Arduino IDE with common instalation for esp32 boards. Latest version of core is 3.0.1? My project uses WiFiClientSecure.h library, which probably causing this problem.
